pom.xml

```diff
@@ -3,15 +3,15 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>4.32</version>
+		<version>4.81</version>
 	</parent>
 
 	<artifactId>robot</artifactId>
 	<version>3.5.2-SNAPSHOT</version>
 	<name>Robot Framework plugin</name>
 	<description>Shows Robot Framework test results in project</description>
 	<packaging>hpi</packaging>
-	<url>https://github.com/jenkinsci/robot-plugin/blob/master/doc/README.md</url>
+	<url>https://github.com/jenkinsci/robot-plugin</url>
```

Comment:
This `url` needs to point to the plugin `README.md` file in order to be automatically uploaded to https://plugins.jenkins.io/robot/. Since there is no readme file in the root of the repository we [need to specify the full path](https://www.jenkins.io/doc/developer/publishing/documentation/#documenting-plugins) for the `README.md` file.

Reply:
Thanks for the correction.  Fixed in 8e42ef2a1eae1953670cb47c281a72179981a252